### ISSUE TITLE
arbitrary_image_stylization not in release

### ISSUE THREAD

**Issue by gar1t** [tensorflow/magenta#1087]
Is there a reason that this model isn't in the current release?

https://github.com/tensorflow/magenta/tree/master/magenta/models/arbitrary_image_stylization

```
$ pip show magenta-gpu
Name: magenta-gpu
Version: 0.3.6
...
$ pip show -f magenta-gpu | grep arbitrary_image_stylization
<BLANKLINE>
```

**Comment by cghawthorne** [comment]
No particularly good reason, just not something we've gotten around to.

@fredbertsch @golnazg is this something you'd be interested in doing?

We'd also be happy to accept a PR. :-)

**Comment by gar1t** [comment]
I can confirm that it works. I suspect it's just a matter of plugging it into the Bazel release machinery gizmo and... voilà!

**Comment by michaelhaessig** [comment]
@gar1t it would be awesome if you could share what additions/changes are needed in the Bazel build configs in order to include arbitrary_image_stylization as a binary in the pip package.

**Comment by gar1t** [comment]
It just needs to land as a module. The only thing I'm doing it grabbing the source from GitHub! :)

I believe this all stems from the use of Bazel to build Python apps, which otherwise would be quite happy including packages via setuptools `find_packages`. I could dig into the Bazel requirements, but I imagine whoever setup this build configuration for magenta would know instantly what to do to make sure this package landed in the distribution.

**Comment by michaelhaessig** [comment]
yes thats what i seems to be the case, i would also prefer not having to dig into the build process :) 

thanks for the fast response !  i also grabbed the source but when i executed the code via
pyhton ./arbitrary_image_stylization_evaluate.py ..  it failed to load the 'from magenta.models.arbitrary_image_stylization' part, it seems that the magenta module code is somehow not loaded in the env.  i've tried the provided docker container which should already have magenta installed, do you have any quick pointers what could be the mistake im making ? thanks again !

**Comment by cghawthorne** [comment]
I think this was addressed in #1148.
